deeplearning4j-core/src/main/java/org/deeplearning4j/datasets/datavec/RecordReaderDataSetIterator.java

```diff
@@ -29,6 +29,8 @@
 import org.datavec.api.records.reader.RecordReader;
 import org.datavec.api.records.reader.SequenceRecordReader;
 import org.datavec.api.writable.Writable;
+import org.datavec.common.data.NDArrayWritable;
```

Comment:
Actually I didn't add this row. Just change package from api.writable  to common.data